uc-catalog-cloning/example.ipynb

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
do you want this to be invoked as a script or as a notebook?

Reply:
Doesn't matter from the practical point of view, but I'd prefer as a notebook